frontend/interfaces/vulnerability.ts

```diff
@@ -1,3 +1,19 @@
+import PropTypes from "prop-types";
+
+export default PropTypes.shape({
+  cve: PropTypes.string,
```

Comment:
Free it's only `cve` and `details_link`

Reply:
"vulnerabilities": [
                    {
                        "cve": "CVE-2022-40023",
                        "details_link": "https://nvd.nist.gov/vuln/detail/CVE-2022-40023",
                        "cvss_score": 7.5,
                        "epss_probability": 0.00136,
                        "cisa_known_exploit": false,
                        "cve_published": "2022-09-07T13:15:00Z",
                        "cve_description": "Sqlalchemy mako before 1.2.2 is vulnerable to Regular expression Denial of Service when using the Lexer class to parse. This also affects babelplugin and linguaplugin.",
                        "resolved_in_version": "1.2.2"
                    }
                ],





[2:24](https://fleetdm.slack.com/archives/C06GKRDPSUT/p1707247495793919)
^^ /hosts/:id: